controls/opensearch.sp

```diff
@@ -0,0 +1,44 @@
+locals {
+  opensearch_compliance_common_tags = merge(local.terraform_aws_compliance_common_tags, {
+    service = "AWS/OpenSearch"
+  })
+}
+
+benchmark "opensearch" {
+  title       = "OpenSearch"
+  description = "This benchmark provides a set of controls that detect Terraform AWS OpenSearch resources deviating from security best practices."
+
+  children = [
+    control.opensearch_domain_encrpted_with_kms_cmk,
+    control.opensearch_domain_enforce_https,
+    control.opensearch_domain_security_group_not_set
+  ]
+
+  tags = merge(local.opensearch_compliance_common_tags, {
+    type = "Benchmark"
+  })
+}
+
+control "opensearch_domain_security_group_not_set" {
```

Comment:
we should rename it to > opensearch_domain_use_default_security_group

Reply:
In this query we are checking OpenSearch domain should not use the default security group shall we still change the query name